### PR TITLE
Add news item about KM/C-PH survival analysis tool using the lifelines package

### PR DESCRIPTION
**Pull request by fubar2**
More ToolFactory spawn - see https://github.com/fubar2/lifelines_tool

### CODE REVIEW

File: content/news/2023-08-11-lifelineskmcph/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Installation for testing
+
+The [lifelines tool](https://toolshed.g2.bx.psu.edu/view/fubar/lifelines_km_cph_tool/dd5e65893cb8), owned by fubar, is available for testing, in the main Galaxy Toolshed.
+It is very new and so not suitable for production use yet. Please let me know if it works for you.
+
+### Tool code
+
+The tool code is available for review at the <a href="https://github.com/fubar2/lifelines_tool">github repository</a> where issues should
+be raised when there are problems or suggestions. This is machine generated code, so pull requests don't
```
Comment (bgruening) [inline content/news/2023-08-11-lifelineskmcph/index.md:60]: A bit redundant to the above

File: content/news/2023-08-11-lifelineskmcph/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Also included are partial plots for each covariate like these from the Rossi recidivism lifelines sample data
+used in the tool test.
+
+![C-PH partial plot samples](agepartialrossi.png)
+
+![C-PH partial plot samples](parolepartialrossi.png)
+
+Uses pandas read_csv with tabular delimiters so should work with any tabular data with the required columns - time and status for observations.
+
+Issues to https://github.com/fubar2/lifelines_tool please.
```
Comment (bgruening) [inline content/news/2023-08-11-lifelineskmcph/index.md:49]: Sounds like a bullet point list but not like a sentence.

Comment (fubar2) [reply content/news/2023-08-11-lifelineskmcph/index.md:49]: Thanks for caring :)
Will adjust shortly. Was mostly cut and paste so editing is good....

Comment (fubar2) [reply content/news/2023-08-11-lifelineskmcph/index.md]: @bgruening - I trust your meal at Doyles is excellent.
Items are shortened.... ;)

Comment (fubar2) [reply content/news/2023-08-11-lifelineskmcph/index.md]: @bgruening - less awful now?

**Review by bgruening — APPROVED** [review]
The meal was excellent :)

**Pull request merged**